src/platform/intel/cavs/lib/clk.c

```diff
@@ -30,6 +30,7 @@ static inline void select_cpu_clock_hw(int freq_idx, bool release_unused)
 {
 	uint32_t enc = cpu_freq_enc[freq_idx];
 	uint32_t status_mask = cpu_freq_status_mask[freq_idx];
+	uint32_t count = 20;
```

Comment:
Should be a platform macro (since it may be different on different CAVS platforms).

Reply:
> Should be a platform macro (since it may be different on different CAVS platforms).

Hi @lgirdwood the platform specific stuff is already covered by PLATFORM_DEFAULT_DELAY.

Reply:
ok, but you should have a local macro then.